server/app/views/admin/programs/ProgramPredicateConfigureView.java

```diff
@@ -515,6 +515,9 @@ private DivTag createScalarDropdown(
     } else {
       try {
         scalars = Scalar.getScalars(questionDefinition.getQuestionType());
+        if (questionDefinition.getQuestionType().equals(QuestionType.NAME)) {
+          scalars = ImmutableSet.of(Scalar.FIRST_NAME, Scalar.MIDDLE_NAME, Scalar.LAST_NAME);
```

Comment:
**nitpick:** Can we add a comment that says something like "Suffix is not included in predicates"